wasi/wasi_test.go

```diff
@@ -254,14 +263,30 @@ func TestArgsGetAPIReturnError(t *testing.T) {
 		tc := tt
 
 		t.Run(tc.name, func(t *testing.T) {
-			errno := wasiEnv.args_get(hostFunctionCallContext, tc.argsPtr, tc.argsBufPtr)
-			require.Equal(t, EINVAL, errno)
+			ret, _, err := store.CallFunction("test", "args_get", uint64(tc.argsPtr), uint64(tc.argsBufPtr))
+			require.NoError(t, err)
+			require.Equal(t, uint64(EINVAL), ret[0]) // ret[0] is returned errno
 		})
 	}
 }
 
-func buildHostFunctionCallContext() *wasm.HostFunctionCallContext {
-	return &wasm.HostFunctionCallContext{
-		Memory: &wasm.MemoryInstance{Buffer: make([]byte, wasm.MemoryPageSize), Min: 1},
+func instantiateWasmStore(wasmFile string, wasiEnv *WASIEnvironment) (*wasm.Store, error) {
+	buf, err := os.ReadFile(wasmFile)
+	if err != nil {
+		return nil, err
+	}
+	mod, err := wbinary.DecodeModule(buf)
+	if err != nil {
```

Comment:
since this is a test, pass `t *testing.T` then you can use require.NoErr which is less crufty as so many less if statements

Reply:
Updated as suggested. Thanks.